src/components/Pages/Form.js

```diff
@@ -92,4 +118,25 @@ function FormContainer({
   );
 }
 
+const ErrorMessage = ({formUrl}) => {
+  return (
+    <div>
+      <h2>We are still waiting for your form to load.</h2>
+
+      <h3>Why hasn't it loaded yet?</h3>
+
+      <p>
+        &bull; A plugin could be blocking the form (example: "Privacy Badger")
+        <br/>
+        &bull; It may have failed to load ("404 error")
+      </p>
+      <br/>
+
+      <h3>Visit the form directly</h3>
+
+      <a href={formUrl} target="_blank"> <p>{formUrl}</p> </a>
+    </div>
+  )
+}
+
```

Comment:
I figure if we have more error messages in the future, this can be cleaned up a bit to be more content"y"/ maybe even a content-type!?